source/analysis/REIXS/REIXSXESImageInterpolationAB.cpp

```diff
@@ -303,6 +303,7 @@ void REIXSXESImageInterpolationAB::setInputDataSourcesImplementation(const QList
 	cachedValues_ = QVector<double>(axes_.at(0).size);
 	axisValueCacheInvalid_ = true;
 	cachedAxisValues_ = QVector<double>(axes_.at(0).size);
+	onInputSourceSizeChanged();

```

Comment:
That may be part of the problem, but in general, a slot having less arguments than the signal is okay provided that if there are parameters that they follow the same order (from first to last).  I would have been very surprised if you changing the slot to having an int would have made a difference.